mobile/src/main/java/org/openhab/habdroid/ui/OpenHABPreferencesActivity.java

```diff
@@ -27,20 +27,21 @@
 import android.support.v7.widget.Toolbar;
 import android.view.MenuItem;
 
+import org.openhab.habdroid.BuildConfig;
 import com.loopj.android.image.WebImageCache;
-
 import org.openhab.habdroid.R;
 import org.openhab.habdroid.util.Constants;
 import org.openhab.habdroid.util.MyWebImage;
 import org.openhab.habdroid.util.Util;
 
 import java.security.cert.X509Certificate;
+import java.text.DateFormat;
```

Comment:
The imports introduced here seem unused.

Reply:
Done!